src/erldns_zone_cache.erl

```diff
@@ -362,6 +463,34 @@ verify_zone(_Zone, DnskeyRRs, KeyRRSigRecords) ->
       VerifyResult
   end.
 
+% Sign RRSet
+-spec(sign_rrset(binary(), [dns:rr()], [dns:rr()], [erldns:keyset()]) -> [dns:rr()]).
+sign_rrset(Name, Records, DnsKeyRRs, KeySets) ->
+  lager:debug("Signing RRSet for zone (name: ~p)", [Name]),
+  KeyRRSigRecords = lists:flatten(lists:map(erldns_dnssec:key_rrset_signer(Name, DnsKeyRRs), KeySets)),
+  RRSetRecords = lists:flatten(lists:map(erldns_dnssec:key_rrset_signer(Name, lists:filter(fun(RR) -> (RR#dns_rr.type =/= ?DNS_TYPE_DNSKEY) end, Records)), KeySets)),
+  lager:debug("RRSetRecords: ~p", [RRSetRecords]),
+  lager:debug("KeyRRSigRecords: ~p", [KeyRRSigRecords]),
+  verify_rrset(DnsKeyRRs, KeyRRSigRecords),
+  %RRSetRecords ++ KeyRRSigRecords ++ rewrite_soa_rrsig_ttl(Records, KeyRRSigRecords).
+  RRSetRecords.
+	
+% Verify RRSet
+-spec(verify_rrset([dns:rr()], [dns:rr()]) -> boolean()).
+verify_rrset(DnsKeyRRs, KeyRRSigRecords) ->
+  lager:debug("Verify RRSet"),
+  case lists:filter(fun(RR) -> RR#dns_rr.data#dns_rrdata_dnskey.flags =:= 257 end, DnsKeyRRs) of
```

Comment:
`include/erldns.hrl` includes a macro for `DNSKEY_KSK_TYPE` that you can use here.

Reply:
commit: https://github.com/dnsimple/erldns/commit/ac0506cd6d09897ae14f38b8bdaa474c421abb58 addresses this.